src/contribute/testing.md

```diff
@@ -255,7 +255,7 @@ precommit and postcommit testing.
 ### Precommit
 
 For precommit testing, Beam uses
-[Jenkins](https://builds.apache.org/view/Beam/) and a code coverage tool
+[Jenkins](https://builds.apache.org/) and a code coverage tool
```

Comment:
Should we change this to `https://builds.apache.org/view/A-D/view/Beam/` instead?